libc-test/build.rs

```diff
@@ -1026,6 +1032,9 @@ fn test_solarish(target: &str) {
             // excluded from the tests.
             "getifaddrs" if is_illumos => true,
 
+            // FIXME: Unsound because `restrict`
+            "lio_listio" => true,
+
```

Comment:
What makes this unsound? Having `restrict` should just be a safety precondition to use this unsafe function.

Reply:
There's a more detailed comment above line 2601, for freebsd:
```
// FIXME: Our API is unsound. The Rust API allows aliasing
// pointers, but the C API requires pointers not to alias.
// We should probably be at least using `&`/`&mut` here, see:
// https://github.com/gnzlbg/ctest/issues/68
```

Reply:
Thanks for the context. I'm not sure I necessarily agree with the solution proposed in that issue, but could you change the `FIXME` to `FIXME(ctest#68)` for context?

Reply:
OK. It seems that all `lio_listio` are skipped and each one brings a block of comment like that. I change the comment to the detailed one.